JS/controllers/TableResultsHeaderCtrl.js

```diff
@@ -0,0 +1,31 @@
+(function () {
+    var app = angular.module('TableResultsHeader', ['dropDownDataService', 'capitalizeFilter']);
+
+    app.controller('TableResultsHeaderCtrl',['dropDownData', function (dropDownData) {
+
+        var vm = this;
+
+        vm.resultsCategory = 'individual';
```

Comment:
It looks like this block (lines 8-14) does the same work as the function below, with "individual" pre-loaded as the value. Is this to set the initial default selection? If so you can probably streamline a bit by replacing it with a call to the function.